src/nimblepkg/options.nim

```diff
@@ -80,6 +81,8 @@ Commands:
   init         [pkgname]          Initializes a new Nimble project in the
                                   current directory or if a name is provided a
                                   new directory of the same name.
+               [--vcs:git]        Add git or hg vcs to the new nimble project,
```

Comment:
I would prefer explicit `--git` and `--hg` flags. Also ``git`` shouldn't be default, neither should be default.

Reply:
yeah, i agree with you, i add both `--git` and `--hg` flags and if user inputs more than one flags, the later will overwrite the before ones.